src/main/java/com/example/javamavenjunithelloworld/Hello.java

```diff
@@ -26,6 +26,24 @@ public void setTimes(int times) {
                     MAXIMUM_AMOUNT_OF_TIMES
             ));
         }
+		int i=0;int j=0;
+					int target = -5;
+		int num = 3;
+
+		target =- num;  // Noncompliant; target = -3. Is that really what's meant?
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Was "-=" meant instead? [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2757)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "target". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)